modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java

```diff
@@ -3853,9 +3853,13 @@ public FragmentEntryLinkModel newFragmentEntryLinkModel(
 						_getFragmentComponentInputStream("heading", "css")),
 					_readFile(
 						_getFragmentComponentInputStream("heading", "html")),
-					_readFile("fragment_component_heading_configuration.json"),
-					_readFile("fragment_component_heading_editValue.json"), 0,
-					headingRenderNamespace));
+					_readFile(
+						"fragment_component_data" +
+							"/fragment_component_heading_configuration.json"),
```

Comment:
@slnn Could you double check this and maybe force push again? Usually we break a long string at the position that it reaches 80 columns. Taking a more obvious example in your change:
```
							"fragment_component_data" +
								"/fragment_component_paragraph_editValue." +
									"json")),
```

should rather be
```
							"fragment_component_data/fragment_component_" +
								"paragraph_editValue.json")),
```

And I think the break should be at punctuation if the string have both characters and punctuation, and keep the punctuation at the end of the previous line. So in the above example, as long as it's not exceeding 80 lines, I keep `_` at the end of first line instead of the beginning of the second line.